x/valset-pref/types/msgs.go

```diff
@@ -33,11 +33,6 @@ func (m MsgSetValidatorSetPreference) ValidateBasic() error {
 	totalWeight := sdk.ZeroDec()
 	validatorAddrs := []string{}
 	for _, validator := range m.Preferences {
-		_, err := sdk.ValAddressFromBech32(validator.ValOperAddress)
```

Comment:
Note: this check was moved to `osmoutils/cli_helpers.go`

Reply:
Why did this get moved to cli_helpers? This should be checked at the validate basic level

Reply:
fixed. It was just a repetitive test, i removed it from cli and added it to validation